### DISCUSSION TITLE
Sunsetting the Amazon Smile Sanitized

### DISCUSSION THREAD

**Discussion by DomHeadroom** [leon-cleaning-services/leon#153] [General]
Since February 20th 2023 Amazon Smile has been shutdown by Amazon and if you try to open an amazon.smile the only thing that will happen is a redirect to an amazon.com link with this tag "?redirectFromSmile=1" appended to the link, so the amazon smile sanitizer doesn't have a real purpose anymore.

**Comment by svenjacobs** [comment]
Thanks for pointing this out! I [removed](https://github.com/svenjacobs/leon/pull/156) the sanitizer.
